src/Core/Content/Product/SalesChannel/Listing/Processor/EmptyProcessTrait.php

```diff
@@ -0,0 +1,16 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Content\Product\SalesChannel\Listing\Processor;
+
+use Shopware\Core\Content\Product\SalesChannel\Listing\ProductListingResult;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\System\SalesChannel\SalesChannelContext;
+use Symfony\Component\HttpFoundation\Request;
+
+#[Package('inventory')]
+trait EmptyProcessTrait
+{
+    public function process(Request $request, ProductListingResult $result, SalesChannelContext $context): void
```

Comment:
do we  need this trait? do we actually want this?

For me this is magic which just hides information, why is this a trait at all? 
Can't we implement this method in the abstract classes as empty and not declare it as abstract when in fact it is optional?

That also solves the issue that currently we use one trait for two interfaces, which means we can't change the interfaces separately anymore